common/internal-types/src/protocol.rs

```diff
@@ -97,24 +97,29 @@ pub enum PendingAcknowledgement {
 pub struct TagBloomFilter {
     bloom: Bloom<PacketTag>,
     count: usize,
+    capacity: usize,
 }
 
 impl TagBloomFilter {
-    // Allowed false positive rate. This amounts to 0.01% chance
-    const FALSE_POSITIVE_RATE: f64 = 0.0001_f64;
+    // Allowed false positive rate. This amounts to 0.001% chance
+    const FALSE_POSITIVE_RATE: f64 = 0.00001_f64;
 
-    // Maximum number of packet tags this Bloom filter can hold.
-    // After this many packets, the Bloom filter resets and packet replays are possible.
-    const MAX_ITEMS: usize = 10_000_000;
+    // The default maximum number of packet tags this Bloom filter can hold.
+    // After these many packets, the Bloom filter resets and packet replays are possible.
+    const DEFAULT_MAX_ITEMS: usize = 10_000_000;
 
     /// Returns the current number of items in this Bloom filter.
     pub fn count(&self) -> usize {
         self.count
     }
 
+    pub fn capacity(&self) -> usize {
+        self.capacity
+    }
+
     /// Puts a packet tag into the Bloom filter
     pub fn set(&mut self, tag: &PacketTag) {
-        if self.count == Self::MAX_ITEMS {
+        if self.count == self.capacity {
             warn!("maximum number of items in the Bloom filter reached!");
             self.bloom.clear();
```

Comment:
In a rare case it is then possible to do the entire bloom filter size (4.6GB) of packet tag replays at this point?